components/HackneyDS/Button/index.js

```diff
@@ -12,7 +12,9 @@ export default function Button({
   color,
   target,
   className,
+  disableShadow,
```

Comment:
use className to disable the shadow, we dont want a million props for a button component